src/Common/getRandomName.cpp

```diff
@@ -0,0 +1,15 @@
+#include <Common/getRandomName.h>
+
+namespace DB
+{
+
+String getRandomName(size_t len, char first, char last)
```

Comment:
@amosbird already fixing it here: https://github.com/ClickHouse/ClickHouse/pull/32745